app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Results Page </h1>
+
+<p>Total Number of Quotes: <%= @quotes_total %></p>
+<p>Keyword Used for this Search: <%= @searched_string %></p>
+
+
+<h2>Top 10 Quotes Found</h2>
+  <% i = 0 %>
+  <% @quotes.each do |quote| %>
+  <div id="quote_<%="#{i}" %>">
+    <%= link_to "#{quote.author}", root_path, class: "navbar-brand" %>
+    <p>Author: <%= quote.author %></p>
+    <p>Quote: <%= quote.quote %></p>
+    <p>Categories: <% quote.categories.each do |category| %></p>
+      <p><%= category %></p>
+    <% end %>
+    <br>
+    <br>
+    <% i += 1 %>
+  </div>
+<% end %>
```

Comment:
In your poro, you already are storing the quotes id. So, you can use that quote id instead of `i`.